docs/soroban-cli-full-docs.md

```diff
@@ -160,7 +160,7 @@ Install a WASM file to the ledger without creating a contract instance
 * `--network-passphrase <NETWORK_PASSPHRASE>` — Network passphrase to sign the transaction sent to the rpc server
 * `--network <NETWORK>` — Name of network to use from config
 * `--ledger-file <LEDGER_FILE>` — File to persist ledger state, default is `.soroban/ledger.json`
-* `--source-account <SOURCE_ACCOUNT>` — Account that signs the final transaction. S...          a seceret key alice         an identity 'kite urban.  a seed phrase DEFAULT       Is the key generated with `identity generate --seed 0000000000000000
+* `--source-account <SOURCE_ACCOUNT>` — Account that signs the final transaction. Alias `source`
```

Comment:
Since the multi-line comments don't show up here, should we consider adding this extra info to the long help for `invoke` itself?

Reply:
If we could fork the markdown generator we fix the multiline issue that way.  But save it for a future PR.